src/App.tsx

```diff
@@ -1,9 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const filterByTitle = (movie: Movie) => (
+    movie.title.toLowerCase().includes(query.toLowerCase())
+  );
+  const filterByDescription = (movie: Movie) => (
+    movie.description.toLowerCase().includes(query.toLowerCase())
+  );
+
+  const visibleMovies = moviesFromServer
+    .filter((movie) => filterByTitle(movie) || filterByDescription(movie));
```

Comment:
1. It's now a filter
2. You create this function each time
Would be better move to do like that
also, move `isIncludes` from the component to avoid recreate this function
```suggestion
  const isIncludes = (string: string, substring: string) => {
    const normalizedString = string.toLowerCase();
    const normalizedSubstring = substring.toLowerCase().trim();
    
    return normalizedString.includes(normalizedSubstring);
  };

  const visibleMovies = moviesFromServer.filter(({ title, description }) => (
    return isIncludes(title, query) || isIncludes(description ,query)
  );
```